.github/workflows/cifuzz.yml

```
@@ -0,0 +1,35 @@
+name: CIFuzz
```

I find it useful in workflows to add a `run-name` value that describes what the action does, so it's more understandable in the list of runs in the Actions tab on GitHub. How about the following?


```suggestion
name: CIFuzz
run-name: >-
  Perform fuzz testing on pull request ${{github.event.pull_request.number}}
  by ${{github.actor}}
```